### PR TITLE
Update bookings-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Updated the wordings for FAq queston

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11269

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/bookings/bookings-faq.yml [hunk 1]
```diff
@@ -246,7 +246,7 @@ sections:
       - question: |
           Is it possible to share only one service on the Self-service page at a time?
         answer: |
-          Yes, you can either create separate Bookings calendars for each service, or you can go to the Services tab, edit a service, and at the top of the page you'll see an option to share a URL only for that specific service.
+          Yes, you can either create separate Bookings calendars for each service, or you can go to the Services tab,  and select Service details and you'll see a URL only for that specific service under Service booking page. You can share the URL which will be for that specific service.
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-faq.yml]:
```suggestion
          Yes, you can either create separate Bookings calendars for each service, or you can go to the Services tab and select Service details. You'll see a URL only for that specific service under the Service booking page. You can then share that URL which will be for that specific service.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**